tests/helm/opentelemetry_operator_instrumentation_cr/static/instrumentation.output.yaml

```
@@ -4,9 +4,9 @@ apiVersion: opentelemetry.io/v1alpha1
 kind: Instrumentation
 metadata:
   namespace: ot-operator1
-  name: RELEASE-NAME-sumologic-opentelemetry-operator-instrumentation
+  name: RELEASE-NAME-sumologic-ot-operator-instr
```

Do we need the `-sumologic` part if we already have the RELEASE-NAME part?
```suggestion
  name: RELEASE-NAME-ot-operator-instr
```

---

Oh I guess that comes from the `sumologic.fullname` part...